assets/scripts/segments/info.json

```diff
@@ -792,6 +792,26 @@
           "repeat": "ground--asphalt",
           "left": "markings--lane-left"
         }
+      },
+      "left|sidewalk": {
+        "elevation": 1,
+        "minWidth": 7,
+        "maxWidth": 10,
+        "graphics": {
+          "left": ["bikes--bikeshare-left", "markings--lane-left"],
```

Comment:
What do you think about removing the lane markings when the station is on a sidewalk level?

Reply:
I was considering that when implementing this, was on the fence personally. I decided to leave them in as it looks like the scooter drop zone has white lines. After trying it out I think I like removing the lines entirely for bikeshare!
![Screen Shot 2020-03-11 at 2 18 39 PM](https://user-images.githubusercontent.com/16242142/76449900-46fc9980-63a3-11ea-9e3c-4518662dce43.png)

Reply:
Pushed the change to remove lane markings.